ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Block/SupportsPeras.hs

```diff
@@ -13,59 +13,49 @@
 
 module Ouroboros.Consensus.Block.SupportsPeras
   ( PerasRoundNo (..)
-  , PerasWeight (..)
+  , onPerasRoundNo
   , BlockSupportsPeras (..)
   , PerasCert (..)
-  , PerasCfg (..)
   , ValidatedPerasCert (..)
-  , makePerasCfg
-  , HasPerasCert (..)
-  , getPerasCertRound
-  , getPerasCertBoostedBlock
-  , getPerasCertBoost
-
-    -- * Ouroboros Peras round length
-  , PerasRoundLength (..)
-  , defaultPerasRoundLength
+  , HasPerasCertRound (..)
+  , HasPerasCertBoostedBlock (..)
+  , HasPerasCertBoost (..)
+
+    -- * Convenience re-exports
+  , module Ouroboros.Consensus.Peras.Params
```

Comment:
I don't generally like re-exporting things, but if we apply this criteria we have to apply it in the whole codebase, so let's keep it for now.